src/main/java/ca/team2706/frc/robot/commands/mirrorable/MirroredCommand.java

```diff
@@ -0,0 +1,85 @@
+package ca.team2706.frc.robot.commands.mirrorable;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * A command that can be mirrored in a certain way
```

Comment:
Might be useful to describe what is meant by mirroring and what the use case would be.